tests/providers/amazon/aws/hooks/test_athena.py

```diff
@@ -0,0 +1,172 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import unittest
+from unittest import mock
+
+from airflow.providers.amazon.aws.hooks.athena import AWSAthenaHook
+
+MOCK_DATA = {
+    'query': 'SELECT * FROM TEST_TABLE',
+    'database': 'TEST_DATABASE',
+    'outputLocation': 's3://test_s3_bucket/',
+    'client_request_token': 'eac427d0-1c6d-4dfb-96aa-2835d3ac6595',
+    'workgroup': 'primary',
+    'query_execution_id': 'eac427d0-1c6d-4dfb-96aa-2835d3ac6595',
+    'next_token_id': 'eac427d0-1c6d-4dfb-96aa-2835d3ac6595',
+    'max_items': 1000
+}
+
+mock_query_context = {
+    'Database': MOCK_DATA['database']
+}
+mock_result_configuration = {
+    'OutputLocation': MOCK_DATA['outputLocation']
+}
+
+MOCK_RUNNING_QUERY_EXECUTION = {'QueryExecution': {'Status': {'State': 'RUNNING'}}}
+MOCK_SUCCEEDED_QUERY_EXECUTION = {'QueryExecution': {'Status': {'State': 'SUCCEEDED'}}}
+
+MOCK_QUERY_EXECUTION = {'QueryExecutionId': MOCK_DATA['query_execution_id']}
+
+
+class TestAWSAthenaHook(unittest.TestCase):
+
+    def setUp(self):
+        self.athena = AWSAthenaHook(sleep_time=1)
```

Comment:
Can we do sleep time=0 instead? (`sleep(0)` is valid) - This would probably make the tests run quicker.